sdk/core/Azure.Core/perf/HttpPipelineBenchmark.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+
+namespace Azure.Core.Perf;
+
+[SimpleJob(RuntimeMoniker.Net60)]
+public class HttpPipelineBenchmark
+{
+    private HttpPipeline _pipeline;
+
+    [GlobalSetup]
+    public void SetUp()
+    {
+        ClientOptions options = new BenchmarkOptions
+        {
+            Transport = new HttpClientTransport(new HttpClient(new MockHttpMessageHandler()))
+        };
+
+        _pipeline = HttpPipelineBuilder.Build(options);
+    }
+
+    [Benchmark]
+    public async Task CreateAndSendMessage()
```

Comment:
Could you add the same test for System.ClientModel pipeline?

Reply:
Added

Reply:
Should the S.CM model test be in the same test class so we could compare S.CM to Azure.Core?

Reply:
Good suggestion